clang/include/clang/CIR/Passes.h

```diff
@@ -0,0 +1,19 @@
+#ifndef CLANG_CIR_PASSES_H
+#define CLANG_CIR_PASSES_H
+
+#include "mlir/Pass/Pass.h"
+
+#include <memory>
+
+namespace cir {
+namespace direct {
```

Comment:
```suggestion
namespace cir::direct {
```

Reply:
There will eventually be things in this file that are in the cir namespace, but not in the cir::direct namespace, so I'd prefer to keep it separate.